pkg/scalers/azure_eventhub_scaler.go

```diff
@@ -0,0 +1,256 @@
+package scalers
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"strconv"
+	"time"
+
+	eventhub "github.com/Azure/azure-event-hubs-go"
+	storageLeaser "github.com/Azure/azure-event-hubs-go/storage"
+	"github.com/Azure/azure-storage-blob-go/azblob"
+	log "github.com/Sirupsen/logrus"
+	"k8s.io/api/autoscaling/v2beta1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/metrics/pkg/apis/external_metrics"
+)
+
+const (
+	defaultEventHubMessageThreshold = 10
```

Comment:
Change the default to whatever is used in the event hub trigger.

Reply:
Do you mean that the default should be specified by the user in a Functions app or that there exists a default for event hub triggered Functions?

Reply:
Yes, there should be a default the PAAS trigger uses- lets use that/